src/tools/telephony/check_extension_status.py

```
@@ -51,6 +51,60 @@ def _parse_dial_string_tech(dial_string: str) -> Optional[str]:
+    if cls == "unavailable":
+        return "unavailable"
+    if state_rec.get("role") == "native":
+        return _NATIVE_BUSY_LABEL.get(str(state_rec.get("state", "")).strip().upper(), "busy")
+    return str(state_rec.get("status") or "").strip().lower() or "busy"
+
+
+def aggregate_availability(states: List[dict]) -> dict:
+    labels = [lbl for lbl in (_label_for(s) for s in states) if lbl]
+    if not labels:
+        return {"available": True, "availability_status": "available", "availability_reason": "available"}
+    top = min(labels, key=lambda l: _STATUS_PRIORITY.index(l) if l in _STATUS_PRIORITY else len(_STATUS_PRIORITY))
```

Ruff (0.16.1)

[error] Ambiguous variable name: `l`

(E741)